packages/cli/src/index.ts

```diff
@@ -20,7 +21,7 @@ export async function main(medplumClient: MedplumClient, argv: string[]): Promis
   }
   try {
     const index = new Command('medplum').description('Command to access Medplum CLI');
-    index.version('0.1.0');
+    index.version(MEDPLUM_VERSION ?? '');
```

Comment:
Would `MEDPLUM_VERSION` be undefined or null?

Reply:
It's based on the environment variables

Reply:
Hm, this is a good catch.  Arguably we should fix this in `packages/core/src/client.ts` so that `MEDPLUM_VERSION` is always a string.